store/cockroach/sql_functions.go

```diff
@@ -0,0 +1,10 @@
+package cockroach
+
+import (
+	"github.com/cortezaproject/corteza-server/pkg/ql"
+)
+
+func sqlFunctionHandler(f ql.Function) (ql.ASTNode, error) {
+	// @todo ^^
```

Comment:
why is this empty? does none of the current implementations (my/pg/lite) fit?

Reply:
Added it, yes it does!